docker/ubuntu14.04/Dockerfile

```
@@ -0,0 +1,42 @@
+FROM ubuntu:14.04
+ENV DEBIAN_FRONTEND noninteractive
+
+## Update cache and upgrade image
+RUN apt-get -y update && apt-get -y upgrade && apt-get -y dist-upgrade
+
+## Build environment packages
+RUN apt-get install -qq -y --ignore-missing \
+	apt-utils		\
+	automake		\
+        bc                      \
```

Small indentation nit.
```suggestion
      bc                      \
```